src/aggregator/aggregator/generic_elem.go

```diff
@@ -908,16 +915,15 @@ func (e *GenericElem) processValue(
 			flushForwardedFn(e.writeForwardedMetricFn, forwardedAggregationKey,
 				int64(timestamp), value, prevValue, cState.annotation, cState.resendEnabled)
 		}
-		// only record lag for the initial flush (not resends)
-		if !fState.flushed {
```

Comment:
a nice benefit of the change. we don't have to skip counting aggregations that are being re-flushed.